Src/CSharpier/Program.cs

```diff
@@ -28,14 +30,36 @@ static async Task<int> Main(string[] args)
             bool check,
             bool fast,
             bool skipWrite,
+            bool writeStdout,
             CancellationToken cancellationToken
         ) {
+            string? standardInFileContents = null;
+            if (Console.IsInputRedirected)
+            {
+                var input = new StringBuilder();
```

Comment:
What's with all this getup? Wouldn't it be easier to use `StreamReader.ReadToEnd()`?

Reply:
Yeah that works too. I blame stumbling across a bad answer on stackoverflow.